chromium/debugging-rulesets.js

```diff
@@ -0,0 +1,20 @@
+/* global sendMessage */
+
+"use strict";
+
+window.onload = function() {
+  document.querySelector("form").addEventListener("submit", save_debugging_rulesets);
+
+  sendMessage("get_option", { debugging_rulesets: "" }, item => {
+    document.getElementById("debugging-rulesets").value = item.debugging_rulesets;
+  });
+}
+
+function save_debugging_rulesets(e){
+  e.preventDefault();
+  sendMessage("set_option", { debugging_rulesets: document.getElementById("debugging-rulesets").value }, () => {
+    const saved = document.getElementById("saved");
+    saved.style.display = "block";
+    setTimeout(() => { saved.style.display = "none" }, 1000);
```

Comment:
Why the delay? Should you also erase `doc.getElementById(#debugging-rulesets).value`?

Reply:
This introduces a delay before removing the "Saved" dialogue, since this should only display for a second.

Reply:
We don't want to clear the value of the `textarea`, since this should reflect what's stored in `debugging_rulesets` in the storage API.